tests/test_symbolic_coefficients.py

```diff
@@ -119,6 +119,110 @@ def test_coefficients_w_xreplace(self):
 
         assert expected == str(eq.evaluate.lhs)
 
+    @pytest.mark.parametrize('order', [1, 2, 6, 8])
+    @pytest.mark.parametrize('extent', [1., 10., 100.])
+    @pytest.mark.parametrize('conf', [{'l': 'NODE', 'r1': 'x', 'r2': None},
+                                      {'l': 'NODE', 'r1': 'y', 'r2': None},
+                                      {'l': 'NODE', 'r1': '(x, y)', 'r2': None},
+                                      {'l': 'x', 'r1': 'NODE', 'r2': None},
+                                      {'l': 'y', 'r1': 'NODE', 'r2': None},
+                                      {'l': '(x, y)', 'r1': 'NODE', 'r2': None},
+                                      {'l': 'NODE', 'r1': 'x', 'r2': 'y'}])
+    def test_default_rules_vs_standard(self, order, extent, conf):
+        """
+        Test that equations containing default symbolic coefficients evaluate to
+        the same expressions as standard coefficients for the same function.
+        """
+        def function_setup(name, grid, order, stagger):
```

Comment:
maybe you can pull these private functions out of the test?  might they be useful for other tests too?

Reply:
I think leave them where they are for now, but maybe add a comment to say that they should be pulled outside if another test needs them in the future?